### ISSUE TITLE
Request to merge with asciidoctorj-screenshot

### ISSUE THREAD

**Issue by mojavelinux** [fix/asciidoctor-screenshot#4]
François-Xavier,

Stephan Classen (@sclassen) has continued development of your excellent work on asciidoctor-screenshot as [asciidoctorj-screenshot](https://github.com/sclassen/asciidoctorj-screenshot). In addition to restructuring the project to produce and publish a JAR file, he has also incorporated feedback and suggestions from the Asciidoctor community (most notably addressing #3).

We'd like to graduate asciidoctorj-screenshot to the Asciidoctor organization. However, before doing so, I'd like to see asciidoctor-screenshot and asciidoctorj-screenshot combine efforts. Since asciidoctorj-screenshot builds on the work of asciidoctor-screenshot, there's no code change necessary. We are merely seeking your support and approval.

**Comment by mojavelinux** [comment]
One concern Stephan has is that the history of the project was not preserved. However, we can fix this in one of two ways:
1. Amend the first commit to asciidoctorj-screenshot to set @fix as the author
2. Rebuild the history of asciidoctorj-screenshot to include the history of asciidoctor-screenshot

I'd be willing to help in anyway necessary to achieve one of these goals.

**Comment by fix** [comment]
Hi @mojavelinux I was not aware of this but this really great! Right now I am on holidays with limited connectivity until next week. I am supporting the idea and back any action you judge necessary to grant the project to the organisation.

Thanks to @sclassen

**Comment by sclassen** [comment]
Hi @fix 
It was a pleasure to work on your code. Most of the work was already there and incorporating the changes was easy.
I myself will be gone the rest of the week too. So if nobody has a hurry I would propose to take this up next week.

**Comment by mojavelinux** [comment]
@fix I'm so happy to read your response! You started something truly awesome and I'm super glad to see the magic @sclassen and other members of the Hackergarten team in Basel have worked on it.

I managed to splice the history of asciidoctor-screenshot onto the front of the history of asciidoctorj-screenshot in a local branch. I can force push that into master once the repository is setup at asciidoctor/asciidoctorj-screenshot.

@sclassen Since your fork has the most recent issues, we'll use that repository as the basis. All I need you to do is:
1. visit https://github.com/asciidoctor to accept the invitation to the Asciidoctor organization
2. transfer the asciidoctorj-screenshot repository to the asciidoctor organization (see https://help.github.com/articles/transferring-a-repository/#to-transfer-a-repository-to-another-user-account).

I'll then handle the rest by making you both admins and pushing the revised history.

Welcome AsciidoctorJ Screenshot to Asciidoctor!

**Comment by mojavelinux** [comment]
@fix Once the merge is complete, you can delete this repository (if you want), then fork the asciidoctor/asciidoctorj-screenshot repository. All your commits will be preserved, so nothing will be lost (aside from the outdated issues).

**Comment by sclassen** [comment]
@mojavelinux I initiated the transfer. But it looks like you need to confirm something because I do not have administration rights in the asciidoctor organization

**Comment by mojavelinux** [comment]
@sclassen You were too fast for me :) I needed to update a setting. Give it another try.

**Comment by mojavelinux** [comment]
Here are the steps I used to merge the histories. Probably not the best way, but it got the job done:

``` bash
#!/usr/bin/bash

git clone https://github.com/fix/asciidoctor-screenshot asciidoctorj-screenshot-merge
git remote add sclassen https://github.com/sclassen/asciidoctorj-screenshot
git fetch sclassen
git rm -rf * .gitignore
git commit --date="Thu Dec 10 21:16:09 2015 +0100" --author="Stephan Classen <st.classen@gmx.ch>" -m "refactor into jar project"
git cherry-pick 1b7eff4f33d95938859da7cde17d02dbe9ed2b2d
git cherry-pick 04a7b91d1a700f64f7dba66da43dcfe586130b59
git cherry-pick 665b3e40cd4cd853b7b3b563a8105821460e7dfa
git cherry-pick 9289a7d6e2ff1d6868293b31fb8ed641f4b0a6f2
git cherry-pick bd7fcb07c31eb8ca540034eb4d72e3830c910f9a
git cherry-pick d534e0aa034fd476ef1732dd7f26f7a3f1395fe4
git cherry-pick 6e4b052fafeb55b70950ebc157c3ad214b37da5f
git cherry-pick 5f6fa91caeee07594fddd5df8fcd05d96d32b206
git cherry-pick 29ddf1566a2d481bdbc2123b59328d323cfe5277
git cherry-pick 839524e8d5a50c33631946f5f48c43c692496a64
git cherry-pick 2746b0ad23c821b95b35f0dc1969f800ff1abfd5
git cherry-pick 9e1cbba5a077f8a0f3f7e76f70f856ea7171ab02
git cherry-pick 13e8befd73f3bf5ec7b598c1b2580853db46dd28
git cherry-pick 0ffdb67ac547ae51983fd1ce33667086f1b3b9d6
git cherry-pick e2db9548250b1ea7a05641bc62b05e66b5a5d066
git cherry-pick 4af8911d9615681865bc3fdb8526dcdbe104e9c8
git cherry-pick b8dd6c6f63d79868e2480a1d1f8d50b176ebf2e7
git cherry-pick c165354ee28d177be628c7b6680162f6a01fe79d
git cherry-pick e352e9a433021f12c8490b1ff570fe2bf2411581
git cherry-pick 6c912b4b323a625555d33fcdd2e089bc0e191c3a
git cherry-pick f5353365be03c6e820d72e92632f8bbcae457d31
git cherry-pick df530f2298aee69815d4f2f82d1f66b73b24c700
git cherry-pick 20794a7782410a67d5cb6a9cee9d91d24ead928f
git cherry-pick df0c2b3a22b8efaeb56f5ffa2077d4f7950ff90a
git cherry-pick ce142b3147ac64689cccd1e5d67ca646db88b012
git cherry-pick 52c4702ac066653fdeb96bb71deb2ff76f78fc5b
git cherry-pick dbeaba528f0724884e39c0dd0ec105db8e077463
git cherry-pick f00adcdbd11b09faf7543645b9d72107c2ca5be9
git cherry-pick 0f15e43d3024d555a2ad1d62893017fed2518e31
git cherry-pick f546de05c1f97617ba83d380f47ebcf516ddc1cc
git cherry-pick b8a59bd8c54f5feba4830aad2d98bba29f986a0d
git cherry-pick 32eda4e2fed5ff0161255795983c79e226ac74d2
git cherry-pick 2ab2d723bbfc2d604ec5409f3499548cc3307125
git cherry-pick 646281beb274994fce0dc53707848274cb1aad81
git cherry-pick 1f78694c824210cf5e68f29be9add12f1386d612
git cherry-pick 287bd9c748ad210dbad2d9bfa4ec2988583c82aa
git cherry-pick 22f33e14c89ea82ab96650cdd1b4149f6d0adee3
git cherry-pick e382928061540095a3d2a41d688323fca4387f29
git cherry-pick b50b703c16542c5b2d2b2ee2fcdf5b523a94cd10
git cherry-pick 8e7fed38fa4c15884ec1c9388c52d4998a8b7a58
git cherry-pick b478c59f1d4106a291584957446bb5d65a8b765c
git cherry-pick 22117b5e528ffacbe8d8d7b28ba195a1da6ceda1
git cherry-pick dcb3990d62610e22c63ede113ab7a12903acaa16
git cherry-pick db57beadb185f4ec750f4b273437edd8f6962e17
git cherry-pick 1a2af094cee6e152b55bbc683e073bc8d4eca2d8
git cherry-pick 2584feed32c23ba2e8b4c245533a4cab5f56de0b
git cherry-pick 2a02819b8426d6c3a4878bd5719df360fba806b4
git cherry-pick 68ae4b12d5c16a77e9bac8d8f1d2bfb02d979f00
git cherry-pick 603752b74dd7548c2594bfd1d3d78108bb3f338a
git cherry-pick c8847e57e93395c02036a8437deb4d4e6f602850
git cherry-pick 106abd2f7c638e46e78652c9176139946ba2ac31
git rebase -i f4e902f90c5dfe2f70dc2c617cbe0d7ab3ff3357
```

I then squashed the "refactor into jar project" commit with the next two commits (and did some other minor grooming).

**Comment by sclassen** [comment]
Transfer in progress :+1:

**Comment by sclassen** [comment]
@mojavelinux the splicing of the two repos looks good. Thanks for doing this. I know it is not the most rewarding job :smile:

**Comment by mojavelinux** [comment]
Thanks @sclassen! I'm still working on cleaning up the history. When I pushed I realized the dates didn't get set right, so I'm applying some fixes. Stay tuned.

**Comment by mojavelinux** [comment]
Okay, I think we're all set. I had to use the GIT_COMMITTER_\* environment variables to preserve the commit author and date. It wasn't pretty, but it got the job done. I think it ended up changing the commit hashes, but hopefully that's not a big deal.

See https://github.com/asciidoctor/asciidoctorj-screenshot

**Comment by mojavelinux** [comment]
If you fork the repository, then links to sclassen/asciidoctorj-screenshot will remain intact.

**Comment by mojavelinux** [comment]
One follow-up task may be restore the example on GitHub Pages so that people can see the result without having to download and run it.
